### ISSUE TITLE
Rearrangement schema priorities for v1.2

### ISSUE THREAD

**Issue by javh** [airr-community/airr-standards#146]
Current release tasks (may be updated):

- [x] Improve validate method in python library. See #142.
- [x] Verify python reference library output is correct on Windows. See #138.
- [x] Verify terminology is consistent between the schema documentation and DataRep manuscript. In particularly: (1) the usage of "required", "optional" and "recommended" fields, (2) "gene" and "segment" and (3) how the schema is named ("AIRR Rearrangement schema", "AIRR Rearrangement Data Representation", "Rearrangement TSV", etc).
- [x] Resolve documentation updates for #69 and #108.
- [x] Alter coordinate definitions from 0-based to 1-based in schema and docs after recent vote. 
- [x] Modify R reference library with coordinate change.
- [x] Modify python reference library with coordinate change.
- [x] Add list of software and databases that support the standard to the docs site.

**Comment by schristley** [comment]
Would be nice to have a link back to the github site from the docs.

**Comment by javh** [comment]
The "Watch" button in the upper left does that. (Options are "Watch", "Star", "Fork" or "Follow": https://ghbtns.com.)

**Comment by schristley** [comment]
ah ok, good to know, thought that was to watch the doc website.

**Comment by javh** [comment]
The R package passes checks on https://win-builder.r-project.org. 

Any objections to submit the v1.2.0 package to CRAN? We can probably safely wait until tomorrow and still meet the manuscript deadline.

**Comment by schristley** [comment]
Fine with me, go for it. I would not be too worried about trying to perfect the software, the schema is set for v1.2 so if we have to go through a v1.2.1, v1.2.2, etc., for minor software fixes, it's no big deal.

**Comment by javh** [comment]
Okay, sounds good.

BTW - Did anyone get a chance to look over the textual changes to the schema descriptions and docs? I proofed them, but another pair of eyes and opinions would be nice.

That annoying R Travis issue popped up in master again. Nothing changed though - the PR branch worked before merging. The `rearrangement_updates` branch basically fixed itself once a rebuild of the image happene. I made changed to the Travis config, but I'm pretty certain all that did was confuse Travis to the point it rebuilt the container.  I can try something similar again... or just wait it out.
